.gitignore

```diff
@@ -122,3 +122,8 @@ ENV/
 
 # Database file
 *.db
+
+# Eclipse stuff
+.project
+.pydevproject
+.settings
```

Comment:
A lot of these changes are the same as in tweaks? Hopefully they are from the same commits and it won't cause a problem, but no need to double up when you're opening a new branch, just return to develop and branch from there, much safer :)